app/helpers/finance_helper.rb

```diff
@@ -8,4 +8,15 @@ def number_to_pounds(number)
   def float_to_percentage(number)
     number_to_percentage(number * 100, precision: 0)
   end
+
+  def total_payment(breakdown)
+    service_fee = Finance::ECF::ServiceFees.new(service_fees: breakdown[:service_fees]).monthly
+    output_payment = Finance::ECF::OutputPayments.new(output_payments: breakdown[:output_payments]).subtotal
+    other_fees = breakdown[:other_fees].map { |other_fee| Finance::ECF::OtherFeeRow.new(other_fee: Finance::ECF::OtherFee.new(other_fee)).subtotal }.inject(&:+)
```

Comment:
`Finance::ECF::ServiceFees` (and others) are ViewComponents, and I really don't like the idea of using them as data objects :/ Can we use the hashes of relevant parts in breakdown to compute the fees?